makefiles/host/rootfs.mk

```diff
@@ -56,3 +56,15 @@ build/nctrl/target/release/nctrl: build/nctrl/.copy_stamp
 	FUSE_CROSS_STATIC_PATH=./thirdparty/ \
 	FUSE_CROSS_STATIC_LIB=fuse \
 	cargo build --release --target=armv7-unknown-linux-musleabihf
+
+build/root.fs/opt/openocd: build/root.fs/.base_install
+	@mkdir -p $(@D)
+	rm -rf $@
+	git clone --depth 1 https://repo.or.cz/openocd.git $@
```

Comment:
We probably want to atleast use a specific commit, maybe even a tagged version?

Reply:
Also why are you not using the official repository on sourceforge?

Reply:
the last tagged version is like 3 years old :(

Reply:
True, then we should atleast pin a specific commit.